hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(i.s0.l()));
+}
```

Comment:
Oh yeah, I missed `ub()`. That'll do the trick. Now lets get rid of MSKTOP altogether...in another PR of course. :-)